manifests/charts/gateways/istio-ingress/values.yaml

```diff
@@ -87,6 +88,8 @@ gateways:
       mountPath: /etc/istio/ingressgateway-ca-certs
 
     customService: false
+    revisionService: true
```

Comment:
conceptually, should we default `false` for revisionService?  Seems most time you just have 1 service, and no need to revision it.

wonder if it is easier to specify:

serviceRevision: default
deploymentRevision: default (or a specific version)

Reply:
morning @linsun. I want to thank you for this suggestion. This has really brought the RFC together - I was a bit stuck until rereading the comments and thinking through your suggestion in the RFC. I didn't use this exact suggestion, although without it, the RFC would have been jammed up. Nice collaboration!